frontend/src/component/project/Project/PaginatedProjectFeatureToggles/AvatarCell.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { vi } from 'vitest';
+import { AvatarCell } from './AvatarCell';
+
+test("the filtering button should have aria-disabled='true' when the user id is 0", async () => {
+    const Cell = AvatarCell(() => {});
+    render(<Cell row={{ original: { createdBy: { id: 0, name: '' } } }} />);
+    const button = await screen.findByRole('button');
+    expect(button).toHaveAttribute('aria-disabled', 'true');
+});
+
+test("the filtering button should have aria-disabled='false' when the user id is not 0", async () => {
+    const Cell = AvatarCell(() => {});
+    render(<Cell row={{ original: { createdBy: { id: 1, name: '' } } }} />);
+    const button = await screen.findByRole('button');
+    expect(button).toHaveAttribute('aria-disabled', 'false');
+});
+
+test('the onAvatarClick function should not be called when the user id is 0', async () => {
+    const callBack = vi.fn();
```

Comment:
weird camel casing. callback is more common

Reply:
Yeah, you're right. Don't know why I did that. Will fix 😅